.gitmodules

```diff
@@ -10,3 +10,7 @@
 [submodule "its/sources/kubernetes-client-go"]
 	path = its/sources/kubernetes-client-go
 	url = https://github.com/kubernetes/client-go
+[submodule "build-logic"]
+	path = build-logic
+	url = https://github.com/SonarSource/cloud-native-gradle-modules
+	branch = SONARGO-52
```

Comment:
Temporary, until https://github.com/SonarSource/cloud-native-gradle-modules/pull/1 is merged